tests/audit-tests/run-tests.sh

```diff
@@ -54,14 +54,21 @@ if [ "$DRIVER" == "http://selenium-hub:4444/wd/hub" ]; then
 
 else
   echo "environment variable DRIVER exists but is not the selinium hubs service name. using the default localhost variant"
-  cd /
+#  cd /
   echo "running selenium using chromedriver"
   export DRIVER=$DRIVER
-
 fi
+export CFN_AppURL="https://di-auth-stub-relying-party-staging-app.london.cloudapps.digital/"
```

Comment:
are these urls are the same for all the environments that run the tests?

Reply:
These tests run only in staging env